src/test/java/com/megazone/goop/racingcar/StringSumCalculatorTest.java

```diff
@@ -0,0 +1,76 @@
+package com.megazone.goop.racingcar;
+
+import org.junit.jupiter.api.*;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class StringSumCalculatorTest {
+
+    private static StringSumCalculator calc;
+
+    @BeforeAll
+    static void initTest() {
+        calc = new StringSumCalculator();
+    }
+
+    @Test
+    @DisplayName("Null or empty string sum")
+    void splitAndSumNullTest() throws Exception {
+        int sumOfNull = calc.splitAndSum(null);
+        assertThat(sumOfNull).isZero();
+
+        int sumOfEmpty = calc.splitAndSum("");
+        assertThat(sumOfEmpty).isZero();
+    }
```

Comment:
`@ParameterizedTest`를 통해 간결하게 작성이 가능합니다.

```suggestion
    @ParameterizedTest
    @NullAndEmptySource
    @DisplayName("Null or empty string sum")
    void splitAndSumNullTest(String text) throws Exception {
        int sumOfNull = calc.splitAndSum(text);
        assertThat(sumOfNull).isZero();
    }
```